splicemachine/spliceengine
# DB-11610/DB-11267 fix sysvw.SYSCOLPERMSVIEW, add more SYSVW views

## Short Description
Fixing issues with SYSVW.SYSCOLPERMSVIEW, SYSCONGLOMERATES, SYSDEPENDS, SYSSEQUENCES

## Long Description

### DB-11610
Fix NPE when performing `select * from SYSVW.SYSCOLPERMSVIEW`.
Reason was an overlooked null check in another code change and the use of a UserType in the view (changed this to a CAST to LONG VARCHAR).

### DB-11267
Currently some SYS.XXX tables can not be accessed from JDBC shells due to a marshalling error because some columns are UserTypes. Since we use those tables internally, we can't just change the column types, but we can creates views that cast those types on server side to char/varchar.

- sys.SYSCONGLOMERATES   -> SYSVW.SYSCONGLOMERATES
- sys.SYSDEPENDS         -> SYSVW.SYSDEPENDS
- sys.SYSSEQUENCES       -> SYSVW.SYSSEQUENCES

Both need an upgrade script so we combined this into one PR.

## How to test
DB-11610: Execute `select * from SYSVW.SYSCOLPERMSVIEW` from a sqlshell not in spliceengine directory.
DB-11267: Check the added views.
